### PR TITLE
[1LP][RFR] Fixed catalog item for cloud providers

### PR DESCRIPTION
**Pull request by sshveta**
{{pytest: -v  --long-running --use-provider azure --use-provider rhos11 cfme/tests/ssui/test_ssui_myservice.py }}

====Couple of fixes====
1) Fixed catalog item for cloud providers and some views in myservice page .
2) Also marked tests with a BZ .
3) Corrected some views
4) Added wait_for in some methods because of long loading time

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 1884a9e2e32e9a19128af13c2b2d1eface313a4e:

`cfme/fixtures/service_fixtures.py`:
- :bangbang: Line 41:24: **E124** *closing bracket does not match visual indentation*
- :bangbang: Line 44:28: **E124** *closing bracket does not match visual indentation*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by mkoura** [comment]
No test ran, I believe you need to add ``--long-running``. Please run also relevant cloud provider test.

File: cfme/services/myservice/ssui.py [hunk 1]
```diff
@@ -75,20 +75,20 @@ class EditMyServiceView(ServiceEditForm):
     @property
     def is_displayed(self):
         return (
-            self.in_myservices and
+            self.name.is_displayed and
```
Comment (rlbabyuk) [inline cfme/services/myservice/ssui.py]: Question: Why in_myservices was replaced?

File: cfme/tests/ssui/test_ssui_myservice.py [hunk 2]
```diff
@@ -14,6 +14,7 @@
 from cfme.utils.providers import ProviderFilter
 from cfme.utils.wait import wait_for
 from cfme import test_requirements
+from cfme.utils.blockers import BZ
```
Comment (rlbabyuk) [inline cfme/tests/ssui/test_ssui_myservice.py]: Required: Please sort imports

File: widgetastic_manageiq/__init__.py [hunk 3]
```diff
@@ -851,6 +851,11 @@ def list_click(self):
 
             return self.list_item.click()
 
+        def is_displayed(self):
+            """Clicks the list item with this name."""
```
Comment (rlbabyuk) [inline widgetastic_manageiq/__init__.py]: Required: docstring is not related to what method is doing

File: cfme/services/myservice/ssui.py (hunk 1 above)
Comment (sshveta) [reply cfme/services/myservice/ssui.py]: Edit service opens a new window on top of the page so in_myservice is False .

**Comment by sshveta** [comment]
5.8 two test case failures are sporadic failures because of selenium bug where it doesn't read catalog name .
@kedark3 : can you please comment on console test failure in 5.8

**Comment by kedark3** [comment]
@sshveta that also is a sporadic selenium failure I believe. I have seen it before, only in PRT, not jenkins or locally.

File: cfme/fixtures/service_fixtures.py [hunk 4]
```diff
@@ -34,16 +35,22 @@ def create_catalog_item(provider, provisioning, vm_name, dialog, catalog, consol
...
+            'environment': {'host_name': {'name': host},
+                            'datastore_name': {'name': datastore}},
+            'network': {'vlan': partial_match(vlan)},
+        }
+    else:
+        provisioning_data = {
+            'catalog': {'vm_name': vm_name},
+            'properties': {'instance_type': partial_match(provisioning['instance_type']),
+                           'guest_keypair': provisioning['guest_keypair'],
+                           'boot_disk_size': provisioning['boot_disk_size']},
```
Comment (lkhomenk) [inline cfme/fixtures/service_fixtures.py]: REQUIRED: consider using  `provisioning.get('boot_disk_size', None)` - e.g. boot_disk_size is GCE specific and Azure doesn't have guest_keypair

File: cfme/fixtures/service_fixtures.py [hunk 5]
```diff
@@ -34,16 +35,22 @@ def create_catalog_item(provider, provisioning, vm_name, dialog, catalog, consol
...
+            'environment': {'availability_zone': provisioning['availability_zone'],
+                            'cloud_network': provisioning['cloud_network']}
+        }
 
     if provider.type == 'rhevm':
         provisioning_data['catalog']['provision_type'] = 'Native Clone'
```
Comment (lkhomenk) [inline cfme/fixtures/service_fixtures.py]: OPTIONAL: this could be in yamls and retrieved by .get

Comment (lkhomenk) [reply cfme/fixtures/service_fixtures.py]: @sshveta I've added azure, for rhos it just skips on 5.9. 
Are these test supposed to run against all providers?

Comment (sshveta) [reply cfme/fixtures/service_fixtures.py]: yes it is supposed to run on all providers

**Comment by sshveta** [comment]
One failure in 5.8 , @kedark3 commented that is sporadic.

**Pull request merged**